zcash_extras/src/wallet.rs

```diff
@@ -0,0 +1,268 @@
+//! Functions for scanning the chain and extracting relevant information.
+use std::fmt::{Debug, Display};
+
+use zcash_primitives::{
+    consensus::{self, BranchId, NetworkUpgrade},
+    memo::MemoBytes,
+    sapling::prover::TxProver,
+    transaction::{
+        builder::Builder,
+        components::{amount::DEFAULT_FEE, Amount},
+        Transaction,
+    },
+    zip32::{ExtendedFullViewingKey, ExtendedSpendingKey},
+};
+
+use crate::WalletWrite;
+use zcash_client_backend::{
+    address::RecipientAddress,
+    data_api::{error::Error, ReceivedTransaction, SentTransaction},
+    decrypt_transaction,
+    wallet::{AccountId, OvkPolicy},
+};
+
+pub const ANCHOR_OFFSET: u32 = 10;
+
+/// Scans a [`Transaction`] for any information that can be decrypted by the accounts in
+/// the wallet, and saves it to the wallet.
+pub async fn decrypt_and_store_transaction<N, E, P, D>(
```

Comment:
As I can see, the original `decrypt_and_store_transaction` is not used in the lib and Komodefi code. Can this be removed?

Reply:
removed

Reply:
@borngraced It is still there

Reply:
it's removed now